components/series/SeriesTable.vue

```diff
@@ -77,13 +77,8 @@
         header-class="front-stack-layer"
         cell-class="is-vcentered">
         <div class="image is-48x48 mb-2">
-          <b-image
-            v-if="!isLoading"
-            :src="props.row.image"
-            :alt="props.row.name"
-            ratio="1by1"
-            rounded />
-          <NeoSkeleton :active="isLoading" circle width="48px" height="48px" />
+          <BasicImage :src="props.row.image" :alt="props.row.name" rounded />
+          <!-- <NeoSkeleton :active="isLoading" circle width="48px" height="48px" /> -->
```

Comment:
Will the skeleton work as before? If so, let's remove this comment line.

Reply:
yup it does work see screenshot in PR description